scene.js

```diff
@@ -58,8 +59,8 @@ class Scene {
 
         // set up the camera position and view matrix
         var cameraPos = [-translate * Math.sin(roll) * Math.cos(-pitch),
-            -translate * Math.sin(-pitch),
-            translate * Math.cos(roll) * Math.cos(-pitch)];
+                         -translate * Math.sin(-pitch),
+                          translate * Math.cos(roll) * Math.cos(-pitch)];
```

Comment:
I don't think this project is going to officially embrace Cesium's or any other coding style anytime soon.  But without a published coding style, we should refrain from changes to styling on existing code.  It creates churn and merge conflicts, and there's been no discussion or agreement that the new style is better.  So generally, my advice is leave the old styling alone.